ext/redhat/pxp-agent.init

```diff
@@ -46,15 +46,10 @@ start() {
 
 stop() {
     echo -n $"Stopping PXP agent: "
-    if [ "$USEINITFUNCTIONS" = "true" ]; then
-        killproc $pidopts $exec
```

Comment:
The sshd implementation also does killproc first. I think we also lose the `[  OK  ]` message from `service` without this.